### PR TITLE
cacafire, cacaview, cacademo, cacaclock: add pages

### PR DESCRIPTION
**Pull request by MIfoodie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
- 0.99.beta19-2.2ubuntu4 amd64

### CODE REVIEW

**Comment by MIfoodie** [comment]
I hope this PR doesn’t get buried :(

File: pages/linux/cacademo.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# cacademo
+
+> Displays a random ASCII art animation.
```
Comment (tricantivu) [inline pages/linux/cacademo.md:3]:
```suggestion
> Display a random ASCII art animation.
```

File: pages/linux/cacafire.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# cacafire
+
+> Displays a animated ASCII fire.
```
Comment (tricantivu) [inline pages/linux/cacafire.md:3]:
```suggestion
> Display an animated ASCII fire.
```

File: pages/linux/cacaview.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# cacaview
+
+> Display a image in PMN format.
```
Comment (tricantivu) [inline pages/linux/cacaview.md:3]:
```suggestion
> Display an image in PMN format.
```

File: pages/linux/cacaview.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://packages.debian.org/sid/caca-utils>.
+
+- Display a image:
```
Comment (tricantivu) [inline pages/linux/cacaview.md:6]:
```suggestion
- Display an image:
```

File: pages/linux/cacaclock.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Display the time:
+
+`cacaclock`
+
+- Change the font:
+
+`cacaclock -f {{font}}`
+
+- Change the format using strftime:
```
Comment (tricantivu) [inline pages/linux/cacaclock.md:14]:
```suggestion
- Change the format using an `strftime` format specification:
```

**Review by tricantivu — COMMENTED** [review]
In my opinion, this pull request has value and should not be closed or deleted even if it mentions poop.

Although I cannot think uses for the programs, there are probably some out there. Ricing/UNIX Porn maybe?

**Review by tricantivu — APPROVED** [review]

**Pull request merged**